governance.md

```diff
@@ -1,98 +1,137 @@
+of the board at any one time. If the results of an election result in greater
+than 40% representation, the lowest vote getters from any particular company
+will be removed until representation on the board is less than one-third.
+
+If percentages shift because of job changes, acquisitions, or other events,
+sufficient members of the committee must resign until max one-third
+representation is achieved. If it is impossible to find sufficient members
+to resign, the entire company’s representation will be removed and new
+special elections held. In the event of a question of company membership (for
+example evaluating independence of corporate subsidiaries) a majority of all
+non-involved Governance Board members will decide.
+
+## Committee Responsibilities and Deliverables
 
 The committee MUST:
 
-* Represent a cross-section of interests, not just one company
-* Balance technical, architectural, and governance expertise since its initial mission is the establishment of structure around contributions, community, and decision-making
-* Hold staggered terms, sufficient to ensure an orderly transition of power via elections as designed and implemented by the committee (see below for specific deliverables)
-* Provide designated alternates in cases where quorum is required but not attainable with the current set of members
+* [Represent a cross-section of interests, not just one company](#maximum-representation)
+* Balance technical, architectural, and governance expertise
+* Hold staggered terms, sufficient to ensure an orderly transition of
+power via elections
+* Provide designated alternates in cases where quorum is required but
+not attainable with the current set of members
 * Communicate with the Continuous Delivery Foundation on a regular cadence
 
-## Governance Facilitator Role (optional)
+The committee is responsible for a series of specific artifacts and
+activities:
 
-The governance facilitator role is a non-voting, non-technical advisory position that helps ensure the bootstrapping process is being adhered to, that all steering committee meetings are inclusive, and key milestones toward governance are met.
-The facilitator is a role, meaning it may be occupied by a single individual, or a series of individuals over time.
-The goal of this position is servant leadership for the project, community, and stakeholders.
-The committee may choose to make this a permanent role at their discretion.
-Deliverables may include creation of mailing lists, project tracking boards, governance draft documents, and so forth.
+* The [Code of Conduct](code-of-conduct.md) and handling violations
+* The [Project Communincation Channels](contact.md)
+* The [Contribution Process](process.md) and [Development Standards](standards.md)
+* The [Tekton Mission and Vision](roadmap.md)
+* Select [election officers](#election-officers) to run elections
 
-## Committee Deliverables
+More?
```

Comment:
are we also responsible for processes like TEP? (not sure)

Reply:
I think we should be, at least before we structure this more ? by "responsible", imho, this mainly means taking a decision if there is unresolved/unresolvable conflict on a TEP on wether it should go in or not

Reply:
Added TEPs